test/utils/kcp.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// EnsureAndSwitchWorkspace ensures that the specified workspace exists in kcp, else creates it, and switches to it.
+func EnsureAndSwitchWorkspace(workspacePath ...string) error {
+	expectedWorkspacePath := "root:" + strings.Join(workspacePath, ":")
+	currentWorkspace, err := Run(exec.Command("kubectl", "kcp", "ws", "."))
+	fmt.Printf("Current workspace: %q, expected workspace: %q\n", currentWorkspace, expectedWorkspacePath)
+	if strings.Contains(currentWorkspace, fmt.Sprintf("'%s'", expectedWorkspacePath)) {
+		return err
+	}
```

Comment:
Shouldn't it be something like this, or am I misunderstanding something here?
```suggestion
	currentWorkspace, err := Run(exec.Command("kubectl", "kcp", "ws", "."))
	if err != nil {
		return err
	}
	fmt.Printf("Current workspace: %q, expected workspace: %q\n", currentWorkspace, expectedWorkspacePath)
	if strings.Contains(currentWorkspace, fmt.Sprintf("'%s'", expectedWorkspacePath)) {
		return nil
	}
```

Reply:
Added error check